3MI0800270/2023-03-07/zad1.cpp

```diff
@@ -0,0 +1,38 @@
+#include <iostream>
+#include <fstream>
+using namespace std;
+
+bool write2file(int a, int b, int c) {
+    ofstream file("results.txt");
+    if (!file.is_open()) {
+        cout << "Error!" << endl;
```

Comment:
You can check the idea of [exceptions](https://www.tutorialspoint.com/cplusplus/cpp_exceptions_handling.htm) for handling errors :))